meta-lmp-bsp/recipes-bsp/u-boot/u-boot-ostree-scr-fit/apalis-imx6-sec/boot.cmd

```diff
@@ -0,0 +1,28 @@
+# Default to ixora-v1.1, which is the carrier board we officially support
+## Careful when changing to another dtb as the mmcblk order might differ
+setenv fdt_file_r imx6q-apalis-ixora-v1.1.dtb
+echo "Using ${fdt_file_r}"
+
+# Default boot type and device
+setenv bootlimit 3
+setenv devtype mmc
+setenv devnum 2
```

Comment:
The main diff from the other boot.cmd is devnum right? If we drop that reordering patch (and rely on the optee rpmb dt entry), this would basically be the same.

That way we can probably even remove this file as it will be updated by https://github.com/foundriesio/meta-lmp/pull/312 (to cover the secure boot case, using fiovb).